vllm/model_executor/models/roberta.py

```diff
@@ -45,6 +45,27 @@ def encoder_decoder_weights():
     return encoder_decoder_weights(), ((n, w) for n, w in all_weights2
                                        if not n.startswith("roberta."))
 
+def maybe_apply_jina_weights_remap(weights: Iterable[Tuple[str,
+                                                           torch.Tensor]]):
+    all_weights1, all_weights2 = itertools.tee(weights)
+
+    if 'emb_ln.weight' not in set([w[0] for w in all_weights1]):
```

Comment:
```suggestion
    if 'emb_ln.weight' not in set([w[0] for w in all_weights1]):
```

This exhausts the `all_weights1` iterator which keeps the whole model in memory for each process because of `itertools.tee`. We should try to avoid this.

Reply:
If the original names from jina model that need to be remapped don't have any overlap with the regular roberta weights, I think we can apply the mapping unconditionally.

Reply:
itertools.tee copied from roberta_task_weights_filter above


Actually I don't know why I do this.

Reply:
If the original names from jina model that need to be remapped don't have any overlap with the names of the regular roberta weights, I think we can apply the re-map unconditionally. 

ok